packages/snap-preact/src/Snap.tsx

```diff
@@ -229,9 +229,26 @@ export class Snap {
 				this.logger.setMode(LogMode.DEVELOPMENT);
 				this.logger.warn(`...loading build... '${branchParam}'`);
 
+				//lets try and get the siteId from the current bundle script in case its not the same as the config.
+				let siteId;
+				try {
+					const scriptsrc = document.getElementById('searchspring-context').getAttribute('src');
+					siteId = scriptsrc
```

Comment:
Let's use this expression to also match the path before the 6 digit siteId `.*\/[a-zA-Z0-9]{6}\/`

```
const path = 'https://snapui.searchspring.io/abc123/bundle.js'.match(/.*\/[a-zA-Z0-9]{6}\//);
if(path) {
    const src = `${path}${branchParam}/bundle.js`;
}
```